plugin/evm/orderbook/tx_processor.go

```diff
@@ -22,14 +23,17 @@ var OrderBookContractAddress = common.HexToAddress("0x03000000000000000000000000
 var MarginAccountContractAddress = common.HexToAddress("0x0300000000000000000000000000000000000001")
 var ClearingHouseContractAddress = common.HexToAddress("0x0300000000000000000000000000000000000002")
 
+// var LimitOrderBookContractAddress = common.HexToAddress("0x0300000000000000000000000000000000000000")
+var IOCOrderBookContractAddress = common.HexToAddress("0x0300000000000000000000000000000000000004")
```

Comment:
FYI - To be compatible with hubblenext, this contradct can't be shipped at genesis now. so we'll deploy the contract and then put the address here.